src/customMetadata/GetStartedChecklistItem.NPSP_Settings.md

```diff
@@ -44,7 +44,7 @@
     </values>
     <values>
         <field>Primary_Button_Value__c</field>
-        <value xsi:type="xsd:string">https://power-of-us-hub-doc.s3-us-west-2.amazonaws.com/NPSP_Planning_Guide.pdf</value>
+        <value xsi:type="xsd:string">navItemPage:c__NPSP_Settings</value>
```

Comment:
Based on the previous discussions, shouldn't the `Primary_Button_Type__c` value be "sfdc:new-tab" rather than "link" when it's pointing to a tab?

Reply:
ooo I missing that change after modify the custom metadata file

Reply:
Should the `Has_Link__c` field still be true for this?

Reply:
@fparodi-salesforce See the comment/question above? 
I'm effectively wondering if this is not a link why the HasLink value is true.

Reply:
Yes has a link is https://bit.ly/3mYoskO